### PR TITLE
Enable the ToString wart.

### PR DESCRIPTION
**Pull request by sellout**
(based on #1452, so don’t merge before that)

### CODE REVIEW

File: mongodb/src/main/scala/quasar/physical/mongodb/expression/exprop.scala [hunk 1]
```diff
@@ -82,6 +86,103 @@ object ExprOp {
...
+        case $toUpperF(value) => Cord("$toUpper(") ++ s.show(value) ++ Cord(")")
+
+        case $metaF() => Cord("$meta")
+
+        case $sizeF(array) => Cord("$size(") ++ s.show(array) ++ Cord(")")
+
+        case $arrayMapF(input, as, in) =>
+          Cord("$arrayMap(") ++ s.show(input) ++ Cord(", ") ++ as.show ++ Cord(", ") ++ s.show(in) ++ Cord(")")
+
+        case $letF(vars, in) =>
```
Comment (drostron) [inline mongodb/src/main/scala/quasar/physical/mongodb/expression/exprop.scala]: Been curious, when do you mark as FIXME vs TODO?

**Comment by drostron** [comment]
Did you intend to add the "Enable IsInstanceOf and AsInstanceOf warts." commit to this PR/branch?

**Comment by sellout** [comment]
Heh, I did. I already had a long chain of dependent PRs, and I figured you hadn’t gotten around to reviewing this one yet, so I added a couple more ;)

File: mongodb/src/main/scala/quasar/physical/mongodb/expression/exprop.scala (hunk 1 above)
Comment (sellout) [inline mongodb/src/main/scala/quasar/physical/mongodb/expression/exprop.scala]: Sort of depending on severity. TODO is a way something could be improved whereas FIXME is a way that something is broken. And I use NB as “this is just the way things are.”

**Comment by drostron** [comment]
OK. Nice to see a few more warts being enabled :-)

File: mongodb/src/main/scala/quasar/physical/mongodb/expression/exprop.scala (hunk 1 above)
Comment (drostron) [inline mongodb/src/main/scala/quasar/physical/mongodb/expression/exprop.scala]: Clarification appreciated.

File: CHANGELOG/revision.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- enable the IsInstanceOf wart (and AsInstanceOf partially)
```
Comment (drostron) [inline CHANGELOG/revision.md]: Should include ToString

**Comment by drostron** [comment]
👍🏽. I'll keep an eye on this and merge once predecessors are in place.

**Comment by drostron** [comment]
Some merge conflict resolution is in order.

**Comment by sellout** [comment]
Updated.

**Pull request merged**